### DISCUSSION TITLE
Проблема с FreeBSD

### DISCUSSION THREAD

**Discussion by nikolaygnp** [bol-van/zapret#873] [Q&A]
Возникла проблема. До недавнего времени обход работал на Фри проблем не было.  С недавнего времени что-то изменилось и на Фри обход работает "косо" т.е. открываются картинки, работает превью, но само видео с ютуба не проигрывается, в тоже время на Linux машине с теми же параметрами видео воспроизводится, Может кто сталкивался с подобной проблемой, или есть какие советы?

**Comment by bol-van** [comment]
какие именно параметры и настройки фаервола ? это не pfsense ?
linux и bsd одинаково подключены к сети ?

**Comment by nikolaygnp** [comment]
> Цитата ответ
Подключения идентичные, На Фри часть фаервола что отвечает за обход выглядит так:
"
# zapret
add 100 allow tcp from me to table\(nozapret\) 80,443
add 101 divert 989 tcp from any to table\(zapret\) 80,443 out not diverted not sockarg xmit ix0
#add 102 divert 989 tcp from table\(zapret\) 80,443 to any tcpflags syn,ack in not diverted not sockarg recv ix0
"
Команда запуска :
/opt/zapret/nfq/dvtws --port=989 --user=daemon --daemon --dpi-desync=split2 --dpi-desync-split-seqovl=4 --dpi-desync-split-pos=5

На Линукс машине обход через nfqws  c параметрами --dpi-desync=split2 --dpi-desync-split-seqovl=4 --dpi-desync-split-pos=5 
для настройки просто [install_easy.sh] с указанным выше параметром ком. строки.

**Comment by bol-van** [comment]
Может быть проблема банальна, и нет нужных IP в таблице zapret ?
GGC имеет широкую географию, могут быть не только на ASке гугла.
Линуху могло повезти попасть в ipset , BSD нет. Или на линухе нет фильтрации по IP, а тут есть

**Comment by nikolaygnp** [comment]
Фильтрация действительно есть (таблица "zapret" - набор сеток по 16-й маске, но я пробовал пропускать весь трафик без таблиц эффект тот же.

**Comment by bol-van** [comment]
В linux по умолчанию в конфиге есть еще дурение quic. Здесь этого нет.
В общем случае надо брать 2 дебаг лога и сравнивать.

**Comment by nikolaygnp** [comment]
Спасибо за консультацию - будем пробовать.

**Comment by nikolaygnp** [comment]
Добрый день. Снял логи с обоих систем, на Linux ролики Youtub проигрываются, То что проходит через BSD - картинка и предпросмотр есть но не проигрывается. Может Вы увидите в чем проблема, мне не удалось понять. Логи прикрепляю к сообщению.
[zapretbsd.log](https://github.com/user-attachments/files/17715329/zapretbsd.log)
[zapretlin.log](https://github.com/user-attachments/files/17715330/zapretlin.log)

**Comment by numbereleven-a** [comment]
В логах не шарю, но видно что:
у вас в bsd логах нет обращения к серверам гугл видео, ip и хостнеймов не нашёл
а вот это встречается в линукс логе
	Строка  522: desync profile search for udp target=80.77.169.205:443 l7proto=quic hostname='rr2---sn-4pvgq-gv8e.googlevideo.com'
	Строка  611: desync profile search for tcp target=80.77.169.176:443 l7proto=tls hostname='rr5---sn-4pvgq-gv8s.googlevideo.com'
	Строка  772: desync profile search for tcp target=80.77.169.240:443 l7proto=tls hostname='rr5---sn-4pvgq-gv8l.googlevideo.com'
	Строка 1859: desync profile search for udp target=80.77.169.172:443 l7proto=quic hostname='rr1---sn-4pvgq-gv8s.googlevideo.com'
	Строка 2223: desync profile search for tcp target=188.43.61.19:443 l7proto=tls hostname='rr8---sn-ug5onuxaxjvh-n8vs.googlevideo.com'

Может быть наведёт на мысль почему запрет не видит сетевых обращений

**Comment by nikolaygnp** [comment]
Покопаю в этом направлении тоже, хотя ДНС установлены тоже аналогично почему не резолва надо проверить.

**Comment by nikolaygnp** [comment]
По результатам: покопавшись в фильтрах выяснилось - не все сервера гуглвидео попадали в фильтр обхода пересобрал фильтры и заработало. Спасибо за наводку

**Comment by bol-van** [comment]
Так я выше и написал в самом начале. GGC имеют широкую географию. Они стоят у провайдеров на их собственных AS.
РКН у провайдеров клянчил этот список, чтобы нам всем еще больше нагадить.
А вы считаете, что у вас он точнее, чем у РКН ?

**Comment by numbereleven-a** [comment]
> По результатам: покопавшись в фильтрах выяснилось - не все сервера гуглвидео попадали в фильтр обхода пересобрал фильтры и заработало. Спасибо за наводку

рекомендую прощёлкать кучу всяких видео на трубе с wireshark и фильтром tls.handshake.extensions_server_name contains "googlevideo.com" && !quic

у меня 50 ggc нашлось

**Comment by nikolaygnp** [comment]
Спасибо за советы, сам уже начал перепроверять через RapidDNS по фильтру "googlevideo.com" тоже нашлось не мало.
